app/models/ems_event/automate.rb

```diff
@@ -3,6 +3,8 @@ module Automate
     extend ActiveSupport::Concern
 
     def manager_refresh(sync: false)
+      return if ext_management_system&.supports?(:streaming_refresh)
```

Comment:
Does this call `allow_targeted_refresh?` under the covers?

Reply:
No streaming refresh is different, it means we don't want to queue targeted refreshes for events because with streaming we get the updates direct from the provider.

Later on in this method we check for allowed_targeted_refresh to check if we should queue a full or a targeted, but for streaming (e.g. vmware) we can skip this completely.